CHANGELOG.md

```diff
@@ -91,6 +95,24 @@ Replace any use of this class with the `govuk-tag--grey` class.
 
 This change was introduced in [pull request #2417: Remove deprecated `govuk-tag--inactive class`](https://github.com/alphagov/govuk-frontend/pull/2417).
 
+### Optional changes
```

Comment:
I think there are a couple of other optional changes we need to add? Listed in this doc https://docs.google.com/document/d/1sANEJKi1PQmjwipdxViIz_gx5PJM_rUTfwelfAU8kcc/edit

Reply:
Ah sorry, my bad. I was only using this PR for changes based on user feedback on the ['test' notes](https://docs.google.com/document/d/1zXF8tRgbRJm8863-laNLB9ZcB4AbWdkNI-IGDtCth2U/edit?pli=1#). I can add those other ones in too.

Reply:
@vanitabarrett Just added them now. Atom behaved a bit oddly - I'd edit something and it made the formatting go haywire. Think I've sorted it now though (said in hope).